src/components/home/Home.vue

```diff
@@ -59,3 +55,9 @@ export default {
   },
 };
 </script>
+
+<style>
+.mainContainerHeight {
+  height: calc(100vh - 11rem);
```

Comment:
11rem jeśli to wysokość headera to... do zmiennej :) Ale możesz zostawić, to tak FYI, w kodzie produkcyjnym by nie przeszło